apps/fabric-website/src/components/Site/Site.module.scss

```diff
@@ -35,6 +35,7 @@
   .siteNavScrollWrapper {
     position: sticky;
     top: 0;
+    max-height: calc(100vh - #{$TopNav-height});
```

Comment:
Just a note that this could potentially result in issues for UHF environments due to the extra nav element that gets added on smaller screens:
![image](https://user-images.githubusercontent.com/307118/57171838-51d2bf00-6dcd-11e9-9bba-24babbfcecfd.png)

I know we had a similar one of our own at one point, but not sure if these styles account for both cases.